### DISCUSSION TITLE
Website open source?

### DISCUSSION THREAD

**Discussion by hailwood** [Tailframes/react-components#1] [Q&A] [answered]
Hey, Nice library!

The components are very cool and well crafted.

I was wondering if you had plans to open source the website itself?
Would be pretty neat for me to use to document my custom copy of the components I use internally :)

**Comment by t-guzik** [comment, marked as answer]
Hey @hailwood, thank you so much for your kind words and positive feedback!

While we currently don't have plans to open-source the website itself, I'd be happy to help you with alternative ways to document your custom components. Feel free to reach out if you have any questions or need assistance!
